specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlServer.json

```diff
@@ -1788,6 +1962,68 @@
           "readOnly": true
         }
       }
+    },
+    "DedicatedSQLminimalTlsSettingsPatchInfo": {
+      "type": "object",
+      "title": "Dedicated SQL minimal tls settings patch info",
+      "description": "Dedicated SQL minimal tls settings patch info",
+      "properties": {
+        "minimalTlsVersion": {
+          "type": "string",
+          "description": "minimal tls version"
+        }
+      }
+    },
+    "DedicatedSQLminimalTlsSettingsProperties": {
+      "description": "Properties of a dedicated sql minimal tls settings.",
+      "type": "object",
+      "properties": {
+        "minimalTlsVersion": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The minimal tls version of the sql server."
+        }
+      }
+    },
+    "DedicatedSQLminimalTlsSettings": {
+      "description": "Dedicated Sql Minimal Tls Settings Info",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "location": {
```

Comment:
>location [](http://example.com/codeflow?start=9&length=8)

proxy resources should not have a location generally

Reply:
@pilor : Removed the location in the latest commit. Please check

Reply:
location property is still present

Reply:
@pilor : In the response we also have location property and with proxy resource will not have **location** and **tags** due to which we need to include location. explcitily  In the same file there is existing "EncryptionProtector" definition which also have proxy resource and location property so new changes have been implemented in similar manner